src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,23 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
+    public static final String EXAMPLE = "123, some street, some unit, some postal code";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
```

Comment:
Since you enforced that address is in Block, Street, Unit, Postal Code format, you might like to change your regex to validate it.